src/components/downloads/_downloads.scss

```diff
@@ -1,35 +1,37 @@
 .download {
```

Comment:
In this file I would include `.download__image`, `.download__image-link`, `.download__meta` and `.download__excerpt` inside the `.download` brackets and then replace the "download" in each of them with "&". to fit in with the style of other SCSS files we have which use Sass. Other than that it looks good

Reply:
Nested. Sure no problem.